azkaban-common/src/main/java/azkaban/executor/ExecutorApiClient.java

```diff
@@ -51,17 +52,18 @@
 @Singleton
 public class ExecutorApiClient extends RestfulApiClient<String> {
 
-  final private static Logger logger = Logger.getLogger(ExecutorApiClient.class);
+  final private static Logger logger = LoggerFactory.getLogger(ExecutorApiClient.class);
+  final private static String DEFAULT_TRUSTSTORE_PATH = "keystore";
```

Comment:
Nit: better to keep private first and then final for consistency as well scope should come first.

Reply:
Fixed. Thanks for pointing out.